### PR TITLE
Js once check file path change

### PR DESCRIPTION
**Pull request by jsingle**
@paetling I think this would work...

Fixes https://github.com/gamechanger/dusty/issues/455

### CODE REVIEW

File: dusty/constants.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 RUN_DIR = '/var/run/dusty'
 SOCKET_PATH = os.getenv('DUSTY_SOCKET_PATH', os.path.join(RUN_DIR, 'dusty.sock'))
-FIRST_RUN_FILE_PATH = os.path.join(RUN_DIR, 'docker_first_time_started')
+FIRST_RUN_FILE_PATH = '~/.dusty_first_time_started'
```
Comment (paetling) [inline dusty/constants.py]: no the problem comes on creating RUN_DIR. need to change that.

Comment (jsingle) [inline dusty/constants.py]: Oh yeah I thought it was just a touch

Comment (paetling) [inline dusty/constants.py]: line 40 of command_file.py

File: tests/unit/command_file_test.py [hunk 2]
```diff
@@ -253,7 +250,6 @@ def test_once_commands_blank(self):
         }
         expected = [
             'if [ ! -f {} ]'.format(constants.FIRST_RUN_FILE_PATH),
-            'then mkdir -p {}'.format(constants.RUN_DIR),
```
Comment (paetling) [inline tests/unit/command_file_test.py]: lol

**Comment by paetling** [comment]
this is worth adding to the dusty docs

**Comment by paetling** [comment]
:ghost: :cow:

**Comment by jsingle** [comment]
Something's failing on integration tests but my local boot2docker is broken or something

**Pull request merged**